docs/introduction/faq.md

```diff
@@ -71,19 +71,21 @@ Uploader][uploader], a service that serves files with CORS headers set.
 
 [iosvideo]: https://developer.apple.com/library/iad/documentation/UserExperience/Conceptual/iAdJSProgGuide/PlayingVideosinAds/PlayingVideosinAds.html
 
+[android-touch-bug]: https://bugs.chromium.org/p/chromium/issues/detail?id=178297
```

Comment:
I've been starting to add the link definition above the paragraph where it's used